package.json

```diff
@@ -72,11 +72,13 @@
     "mocha": "^8.1.3",
     "mockdate": "^3.0.2",
     "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
```

Comment:
We already use `concurrently` for building internal dependencies - it might not be necessary to use a different package here?

Reply:
Oh I see now, they've come out of psss - all good then